pom.xml

```diff
@@ -31,7 +31,7 @@
 
         <!-- THIS SHOULD BE KEPT IN SYNC TO VERSION IN CGDS.SQL -->
 		<db.version>2.13.1</db.version>
-
+        <genetable.version>hgnc_v2023.10.1</genetable.version>
```

Comment:
@rajeswari2904 should this be pulled from the SQL instead? Do we need to hardcode it in the pom?

Reply:
I think the hgnc version is a bit different from `db.version`

- the `db.version` indicates the version of the SQL schema. The backend is only compatible with a specific version of this schema
- the genetable version is fine if it differs with the backend code version. No need to hardcode the version in the pom.xml. Whoever loads the gene table data should be setting this version